packages/image_picker/image_picker_platform_interface/lib/src/platform_interface/image_picker_platform.dart

```diff
@@ -186,6 +187,8 @@ abstract class ImagePickerPlatform extends PlatformInterface {
     throw UnimplementedError('getImage() has not been implemented.');
   }
 
+  /// This method is deprecated in favor of [getMultiImageWithOptions] and will be removed in a future update.
```

Comment:
Need to mark the method to be deprecated: `@Deprecated('message')`

Reply:
Using `@Deprecated` annotation causes the `analyze` command to fail in `examples` of Android and iOS packages, because they use local version of the platform interface instead of the one from pub.dev. I guess it could be used after 
1. Merging platform interface changes and publishing them to the Pub repository
2. Updating examples to use new methods

If you don't have any other ideas, I'll just remove the annotations

Reply:
Yes, we don't use `@Deprecated` for this (for the reason described above). @cyanglaz, see discussion in https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-platform-interface-method-parameters